libs/core/build.gradle

```diff
@@ -40,6 +40,8 @@ dependencies {
   api project(':libs:opensearch-common')
 
   api "com.fasterxml.jackson.core:jackson-core:${versions.jackson}"
+  api "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
```

Comment:
Shouldn't this dependency be added to the `:server` module where the parsing logic exists instead of here? Why change the dependencies of this library if no code in this library needs the dependency?

Reply:
Done. I had added it here thinking this project is being imported by multiple places and already has a jackson dependency it might be useful. I have move it to server/build.gradle in my latest commit: [532d13a](https://github.com/opensearch-project/OpenSearch/commit/532d13ac016d24827003694938899d3ca56624af)